.github/workflows/AtlasBuildTest.yml

```diff
@@ -0,0 +1,25 @@
+name: Testing Atlas Build
+
+on:
+    push:
+        paths:
+            - "packages/theming/atlas/**"
+
+jobs:
+    test:
+        name: "Testing Atlas Build"
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: "Checking-out code"
+              uses: actions/checkout@v2
+              with:
+                  submodules: false
+            - name: "Installing dependencies"
+              run: npm install
+            - name: "Running Atlas build"
+              run: |
+                  pwd
```

Comment:
Why do we need to to print the working directory?

Reply:
Actually, it's not necessary. Removed.